src/any-type-field/any-type-field.component.html

```diff
@@ -1,23 +1,25 @@
-<div [ngSwitch]="componentType">
-  <div *ngSwitchCase="'table-list'">
-    <table-list-field [values]="value" [schema]="schema" [path]="path"></table-list-field>
-  </div>
-  <div *ngSwitchCase="'complex-list'">
-    <complex-list-field [values]="value" [schema]="schema" [path]="path"></complex-list-field>
-  </div>
-  <div *ngSwitchCase="'primitive-list'">
-    <primitive-list-field [values]="value" [schema]="schema" [path]="path"></primitive-list-field>
-  </div>
-  <div *ngSwitchCase="'object'">
-    <object-field [value]="value" [schema]="schema" [path]="path"></object-field>
-  </div>
-  <div *ngSwitchCase="'ref'">
-    <ref-field [value]="value" [schema]="schema" [path]="path"></ref-field>
-  </div>
-  <div *ngSwitchCase="'value-change-watcher'">
-    <value-change-watcher [value]="value" [schema]="schema" [path]="path"></value-change-watcher>
-  </div>
-  <div *ngSwitchDefault>
-    <primitive-field [value]="value" [schema]="schema" [path]="path"></primitive-field>
+<div [ngClass]="{disableCursorClass: schema && schema['x_editor_disabled']}">
```

Comment:
`schema.x_editor_disabled` is fine in the template, you don't have to put brackets.